package.json

```diff
@@ -69,5 +70,8 @@
     "ts-jest": "^27.1.4",
     "tsutils": "^3.21.0",
     "typescript": "^4.8.3"
+  },
+  "peerDependencies": {
+    "node": ">=14.21.3"
   }
```

Comment:
I wonder what's the difference between this and

```
  "engines": {
    "node": ">=14.21.3"
  }
```

Reply:
Good question, I wasn't sure about this, so I looked it up closer now.

The [documentation for package.json](https://docs.npmjs.com/cli/v9/configuring-npm/package-json#engines) is quite ambiguous. I was under the impression that the `"engines.node"` field was for applications, but as you suggest, it turns out that it should be used for the `node` dependency as well. I had a look around at various libraries on NPM to see how they treated this problem, and although there's no consensus, the bigger libraries (such as Express) puts `node` in the `engines` field.

So I learned something, thanks for pointing it out 😄